tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "lib": ["esnext"],
     "moduleResolution": "node",
     "strict": true,
-    "target": "ES6"
+    "target": "ES6",
+    "strictPropertyInitialization": false
```

Comment:
Disabling so our models don't need to append `!` to every property. Someone tell me if this is a really bad idea...

Reply:
This disables this check for all class properties in the code base, not just for models. What is the issue with appending `!` (which makes this very explicit)?

Reply:
Yeah, fair point. Not a major issue to be explicit in our model files. Will revert this.

- [ ] TODO: Revert `"scriptPropertyInitialization": false` change